.github/workflows/ci-wordpress.yaml

```diff
@@ -0,0 +1,102 @@
+name: CI-Wordpress
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for deploy wordpress to shared hosting by ansistrano ansible role
+#
+# Original Author: Rostislav Udaltsov
+# Created: 09/19/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      server_host:
+        required: true
+        type: string
+        description: 'The address of the server'
+      ssh_deploy_user:
+        type: string
+        required: true
+        description: 'The username for connect to production server via ssh'
+      environment:
+        required: true
+        type: string
+        description: 'The environment'
+      playbook_path:
+        required: true
+        type: string
+        description: 'The path to the file relative to the repository'
```

Comment:
Not clear description, add info that it is path to ansible playbook within current repo

Reply:
done